pkg/collector/corechecks/cluster/orchestrator/processors/processor.go

```diff
@@ -20,6 +20,8 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 )
 
+var json = jsoniter.ConfigCompatibleWithStandardLibrary
```

Comment:
Could you add a comment on the reasoning of doing this? 

Similar to here: https://github.com/DataDog/datadog-agent/blob/640b93dad327d460a42702d2b3a354c549f49b4d/pkg/collector/corechecks/cluster/orchestrator/transformers/k8s/pod.go#L112-L114